index.html

```diff
@@ -105,7 +105,7 @@ <h1>I'm Zermarden <br>
           </div>
           <div class="card-works-2">
             <div class="snapshot-portfolio">
-              <img class="snp-p-img" src="assets/snapshoot-portfolio.svg" width="295px" height="220px">
+              <img class="snp-p-img" src="assets/snapshoot-portfolio.svg" width="295" height="220" alt="snapshot-portfolio">
```

Comment:
- [ ] Your project images are not displaying because the image file you're referencing is not in the assets folder.